modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm

```diff
@@ -65,5 +65,10 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/storage/toolbox/syndicate = 100,
 	/obj/item/storage/toolbox/infiltrator = 100,
 	/obj/item/toy/sprayoncan = 200,
-	/obj/item/warpwhistle = 5
+	/obj/item/warpwhistle = 5,
+	/obj/item/spear/grey_tide = 1,
+	/obj/item/shadowcloak = 1,
+	/obj/item/clothing/head/helmet/abductor = 1,
+	/obj/item/clothing/shoes/jackboots/fast = 1,
```

Comment:
This neither, there's a reason we limit move-speed boosts, there shouldn't be 'makes you fast as fuck boots' just casually in maints.